ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -431,6 +431,52 @@ def test_torch_instance_addbmm_(
     )
 
 
+# addcdiv_
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="addcdiv_",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=3,
+        large_abs_safety_factor=2.5,
+        small_abs_safety_factor=2.5,
+        safety_factor_scale="log",
+        shared_dtype=True,
+    ),
+    value=st.floats(min_value=-100, max_value=100),
+)
+def test_torch_instance_addcdiv_(
+    dtype_and_x,
+    value,
+    frontend,
+    frontend_method_data,
+    init_flags,
+    method_flags,
+    on_device,
+):
+    input_dtype, x = dtype_and_x
+    assume(not np.any(np.isclose(x[2], 0)))
```

Comment:
We should ideally not use `assume` here, instead use the min and max values in the strategy.

Reply:
I want to avoid division by zero here while testing on both positive and negative values. Can we achieve this without using `assume`?

Reply:
we can set the parameter `abs_smallest_val` larger than 0 to avoid generating 0.